HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthConfiguration.swift

```diff
@@ -125,6 +125,11 @@ public final class GiniHealthConfiguration: NSObject {
                                                                                      placeholderForegroundColor: GiniColor(lightModeColor: UIColor.GiniHealthColors.dark4,
                                                                                                                            darkModeColor: UIColor.GiniHealthColors.light4).uiColor())
     
+    /**
+     Localization configuration
+     */
+    public var localization: GiniLocalization?
```

Comment:
Sorry @motokotomax  I think I've confused you here, I was proposing to call a public property `customLocalization` instead of `localization`